content/updates/2020-07/2-rules-date-format.md

```diff
@@ -0,0 +1,19 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# A new relative date format
+
+This is now possible to filter products/product models based on a **relative date** in the rules engine. This new format is available via YML import only for now, and for these 3 filters: `updated`, `created`, and `date` attribute type.
+
+It was already possible to filter products/product models on their `creation date`, `updated date`, or a `date` attribute type. But it was not enough.
```

Comment:
```suggestion
It's already possible to apply conditions on products/product models based on their `creation date`, `updated date`, or the content of a `date` attribute type. But we wanted to go further, by allowing you to not only filter on **specific dates** but also **relative dates**. With this new functionality, you'll be able to apply an action on all products created/updated since the past X days. 
```